types/fulu/data_column_sidecar.yaml

```diff
@@ -0,0 +1,34 @@
+Fulu:
+  DataColumnSidecar:
```

Comment:
I don't see this used anywhere in the beacon-APIs, are you thinking to introduce a `getDataColumns` endpoint?

Reply:
Cool, seems like this is strictly api related, so I'll remove this

Reply:
> are you thinking to introduce a getDataColumns endpoint?

this seems potentially useful but I would recommend to keep this PR to minimal required changes for peerdas to make it easier to merge